src/Microsoft.AspNet.Diagnostics.Entity/DatabaseErrorPageOptions.cs

```diff
@@ -1,18 +1,15 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using Microsoft.AspNet.Http;
+
 namespace Microsoft.AspNet.Diagnostics.Entity
 {
     public class DatabaseErrorPageOptions
     {
-        public static DatabaseErrorPageOptions ShowAll => new DatabaseErrorPageOptions
-                                                              {
-                                                                  ShowExceptionDetails = true,
-                                                                  ListMigrations = true
-                                                              };
-
         public virtual bool ShowExceptionDetails { get; set; }
-
         public virtual bool ListMigrations { get; set; }
+        public virtual bool EnableMigrationCommands { get; set; }
```

Comment:
Doc comments on at least all the properties, but on the class would be nice too.